package.json

```diff
@@ -62,6 +61,7 @@
   },
   "devDependencies": {
     "@deck.gl/aggregation-layers": "^9.0.30",
+    "@deck.gl/carto": "^9.0.30",
```

Comment:
The dev dependency is only here (after this PR) because we need it in examples...

https://github.com/CartoDB/carto-api-client/blob/main/examples/01-pure-js/pure-js.ts

... and it's a bit hard to the layer sources without that. So it won't be a "circular dependency" for users of either package, I think that should be OK?